src/Standards/BestIt/Sniffs/Commenting/AbstractDocSniff.php

```diff
@@ -346,7 +346,7 @@ private function validateOneLineSummary(): bool
         $summaryPos = $this->getSummaryPosition();
         $nextPossiblePos = $this->loadNextDocBlockContent($summaryPos);
 
-        if ($nextPossiblePos > -1) {
+        if ($nextPossiblePos !== false && $nextPossiblePos > 0) {
```

Comment:
$nextPossiblePos can only be false, 0 or positive integers, right? So how about:
if ($nextPossiblePos)

Reply:
Changed from bool to null/int, adjusted this as well